### PR TITLE
Local storage - Contact

### PR DESCRIPTION
**Pull request by Hamzaoutdoors**
- Test local storage availability 
- Save data with local storage
- Get data every time the user refreshes the page

### CODE REVIEW

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @Hamzaoutdoors and @xJokeer, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: You have used localstorage to store the data. :muscle: 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: js/app.js [hunk 1]
```diff
@@ -188,6 +187,49 @@ contactForm.addEventListener('submit', (ev) => {
...
+  if (!Object.values(data).every((o) => o === '')) {
+    for (let i = 0; i < inputs.length; i += 1) {
+      inputs[i].value = Object.values(data)[i];
+    }
+  }
+};
+displayData();
+
+// Additional features
+const borderDisplay = () => {
```
[the remark this answers is not in the document]
Comment (xJokeer) [reply js/app.js:232]: Can you please check it well? we are using `onchange` 😁

[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply js/app.js:232]: For me it's not working properly, that's why I asked you. I first checked with githack, then by cloning, but the issue still persist.

[the remark this answers is not in the document]
Comment (xJokeer) [reply js/app.js:232]: But I just run it with Githubpages and it works fine, can you please check it  [here](https://hamzaoutdoors.github.io/My_Portfolio/) again?

[the remark this answers is not in the document]
Comment (xJokeer) [reply js/app.js:232]: But we just check it and it's works fine, can you please check it [here](https://hamzaoutdoors.github.io/My_Portfolio/) again?

[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply js/app.js:232]: Looks like it working. :thinking:

[the remark this answers is not in the document]
Comment (Dipeshtwis) [reply js/app.js:232]: Can you please request again. I am really sorry if I did some mistake while reviewing :pray:

**Review by mricanho — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @Hamzaoutdoors @xJokeer  🔥 

Your form works as expected, good job guys👍🏽 

that's how you do it

Now you can merge this branch 🚀 

### 💻 👩🏽‍💻  Happy Coding!  👨🏽‍💻 🖥️

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**